subiquity/common/ubuntu_advantage.py

```diff
@@ -158,7 +158,7 @@ async def get_avail_services(self, token: str) -> List[dict]:
             raise ExpiredUATokenError(token, expires=info["expires"])
 
         def is_avail_service(service: dict) -> bool:
```

Comment:
1) Now that we check for both available && entitled, we probably should adjust the name of the methods to not use the term 'available' - `get_display_services` maybe?  Is there a term we can borrow from u-a for something that is both available and entitled?  Does entitled imply available?
2) The commit message is good material and I'd like to include it as a direct code comment.

Reply:
I agree. The name seems confusing now.
I guess an adequate term would be "enableable" ; which is surprisingly difficult to write. Are you happy with the term "activable" ?
I will include the content of the commit message as soon as we agree on the wording :)

Reply:
Sure, let's do activable.  Thanks!